llvm/tools/llvm-profgen/ProfiledBinary.cpp

```diff
@@ -718,7 +740,8 @@ void ProfiledBinary::disassemble(const ELFObjectFileBase *Obj) {
   }
 
   // Dissassemble rodata section to check if FS discriminator symbol exists.
-  checkUseFSDiscriminator(Obj, AllSymbols);
+  if (auto *ELFObj = dyn_cast<ELFObjectFileBase>(Obj))
+    checkUseFSDiscriminator(ELFObj, AllSymbols);
```

Comment:
Should we support `checkUseFSDiscriminator`(FS-FDO) for COFF?  the function just parses the section names to check if "__llvm_fs_discriminator__" name exists, feels it's independent to the format.

Reply:
Done